models/product-type/src/product-type-draft/generator.ts

```diff
@@ -0,0 +1,17 @@
+import { AttributeDefinitionDraft } from '@commercetools-test-data/attribute-definition';
+import { fake, Generator } from '@commercetools-test-data/core';
+import { TProductTypeDraft } from '../types';
+
+// https://docs.commercetools.com/api/projects/productTypes#producttypedraft
+
+const generator = Generator<TProductTypeDraft>({
+  fields: {
+    name: fake((f) => f.random.words(2)),
+    key: fake((f) => f.lorem.slug()),
+    description: fake((f) => f.random.words()),
+    attributes: fake(() => [AttributeDefinitionDraft.random()]),
+    attributeDefinitions: fake(() => [AttributeDefinitionDraft.random()]),
```

Comment:
This is probably nitpicky, but my understanding is that because this is based on a REST field we should handle configuring this in the graphql transformation 🤔